<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-4</y>
      <width>621</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Special</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>161</width>
        <height>95</height>
       </rect>
      </property>
      <property name="title">
       <string>Fit bounds</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QLineEdit" name="a_min">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>20</y>
         <width>59</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="a_max">
       <property name="geometry">
        <rect>
         <x>97</x>
         <y>20</y>
         <width>59</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>65</x>
         <y>20</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt; a &lt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="b_min">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>45</y>
         <width>59</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="b_max">
       <property name="geometry">
        <rect>
         <x>97</x>
         <y>45</y>
         <width>59</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>65</x>
         <y>45</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt; b &lt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>65</x>
         <y>70</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt; c &lt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="c_max">
       <property name="geometry">
        <rect>
         <x>97</x>
         <y>70</y>
         <width>59</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="c_min">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>70</y>
         <width>59</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>General</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>75</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Delimiter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>45</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>.xlsx, .csv, .txt, ...</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>81</width>
        <height>45</height>
       </rect>
      </property>
      <property name="title">
       <string>Fit</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QComboBox" name="drop_fit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>x^2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>exp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>linear</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sqrt</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QPushButton" name="bttn_load">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>41</y>
        <width>72</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
     <widget class="QComboBox" name="drop_column">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>90</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>x-y-yerr</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>x-y</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>y-x</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>x-xerr-y-yerr</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="bttn_exit">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>70</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>81</width>
        <height>45</height>
       </rect>
      </property>
      <property name="title">
       <string>Color</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QComboBox" name="drop_color">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yellow</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Columns</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>70</width>
        <height>45</height>
       </rect>
      </property>
      <property name="title">
       <string>Isotope</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QLineEdit" name="isotope">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>20</y>
         <width>59</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="delimiter">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>59</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="folder_path">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>45</y>
        <width>421</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bttn_plot">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>70</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Easy plotter: Select a file, change some options and plot. Fit is integrated in the figure's zoom button.</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
